docs/docs/migration.md

```diff
@@ -52,18 +55,34 @@ If you need to change the default build behavior defined for the corresponding b
           url: ${default-url}
 ```
 After renaming, make sure that the places where the name is used refer to the correct artifact.
+</li>
+
+<br>
+<li>
+
+
+The `hdb` builder is not supported by the Cloud MTA Build tool.  You no longer require builder settings for the `hdb` module because the required `npm install --production` command is run by default for this module type.
+
+If you used this builder for other module types, you can repace it with the `npm` builder or use the `custom` builder that runs the `"npm install --production"`command.
+
+
   
+
+</li>
+
 ---
-**NOTE**
-If you try to build the project without these changes, the build tool will identify this and will fail the build with the corresponding errors.
+**NOTE:**
+<b>If you try to build the project without the changes above, the build tool will identify this and will fail the build with the corresponding errors.</b>
 
 ---
 
-* `JSON` files with [service creation parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) or [service binding parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) referenced by `path` property of the correponding entity in the `mta.yaml` are packaged differently into the result MTA archive. 
+<li> 
+
+`JSON` files with [service creation parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/a36df26b36484129b482ae20c3eb8004.html) or [service binding parameters](https://help.sap.com/viewer/65de2977205c403bbc107264b8eccf4b/Cloud/en-US/c7b09b79d3bb4d348a720ba27fe9a2d5.html) referenced by the `path` property of the correponding entity in the `mta.yaml`, are packaged differently into the result MTA archive.  &nbsp;  &nbsp; &nbsp;
```

Comment:
Did you mean "are packaged directly" instead of "are packaged differently"?

Reply:
Paola reviewed the texts yesterday. I meant "differently", I will re-phrase slightly to make my point more clear.